samples/sensor/bme280/src/main.c

```diff
@@ -9,15 +9,27 @@
 #include <zephyr/device.h>
 #include <zephyr/devicetree.h>
 #include <zephyr/drivers/sensor.h>
+#include <zephyr/drivers/sensor_data_types.h>
+#include <zephyr/rtio/rtio.h>
+#include <zephyr/dsp/print_format.h>
 
 /*
  * Get a device structure from a devicetree node with compatible
  * "bosch,bme280". (If there are multiple, just pick one.)
+ *
+ * TODO: Won't work as expected with multiple BME280 devices.
  */
-static const struct device *get_bme280_device(void)
-{
-	const struct device *const dev = DEVICE_DT_GET_ANY(bosch_bme280);
+const struct device *const dev = DEVICE_DT_GET_ANY(bosch_bme280);
+
+SENSOR_DT_READ_IODEV(iodev, DT_NODELABEL(bme280),
+		{SENSOR_CHAN_AMBIENT_TEMP, 0},
+		{SENSOR_CHAN_HUMIDITY, 0},
+		{SENSOR_CHAN_PRESS, 0});
```

Comment:
@teburd Because there's no guarantee DEVICE_DT_GET_ANY will return the same of multiple BME280 devices as DT_NODELABEL, I'm trying to figure out how to get the label / alias from the existing device struct so I can pass it into SENSOR_DT_READ_IODEV. Is there something simple I'm missing?

Reply:
DT_COMPAT_GET_ANY_STATUS_OKAY(bosch_bme280) should return the same token needed here

Reply:
Changed